chapters/annotations.tex

```diff
@@ -2324,6 +2324,16 @@ \subsection{Licensing}\label{licensing}
 \end{example}
 
 
+\section{License Texts}\label{license-texts}
```

Comment:
What do you think, could we also add a better example here (similar as in 18.9.2)?

Reply:
We could, but I think it can wait.